core/templates/pages/exploration-editor-page/modal-templates/exploration-save-modal.template.html

```diff
@@ -1,31 +1,32 @@
-<div class="modal-header">
-  <h3>
-    <span ng-if="isExplorationPrivate">Save Draft</span>
-    <span ng-if="!isExplorationPrivate">Publish Changes</span>
-  </h3>
-</div>
-<div class="modal-body oppia-long-text">
-  <p>
-    <span ng-if="isExplorationPrivate">(Optional)</span>
-    Please enter a brief description of what you have changed:
-    <textarea rows="3" cols="50" maxlength="1000" ng-model="commitMessage" focus-on="saveChangesModalOpened" class="oppia-commit-message-input protractor-test-commit-message-input" autofocus></textarea>
-  </p>
-  <button class="btn btn-secondary" ng-click="onClickToggleDiffButton()">
-    <span ng-if="!showDiff">Show Changes</span>
-    <span ng-if="showDiff">Hide Changes</span>
-  </button>
-
-  <version-diff-visualization ng-if="showDiff" diff-data="diffData"
-                              earlier-version-header="earlierVersionHeader"
-                              later-version-header="laterVersionHeader">
-  </version-diff-visualization>
-</div>
-<div class="modal-footer">
-  <button class="btn btn-secondary" ng-click="cancel()">Cancel</button>
-  <button class="btn btn-success protractor-test-save-draft-button" ng-disabled="!isExplorationPrivate && !commitMessage" ng-click="confirm(commitMessage)">
-    <span ng-if="isExplorationPrivate">Save Draft</span>
-    <span ng-if="!isExplorationPrivate">Publish Changes</span>
-  </button>
+<div ng-class="{'oppia-save-exploration-wide-modal':showDiff}">
```

Comment:
Just to check, can you include a screenshot of this modal with your changes? Thanks!

Reply:
Oops, I missed the screen recording. The previous comment is not relevant.